homeassistant/components/samsungtv/media_player.py

```diff
@@ -107,14 +104,13 @@ def __init__(
         self._attr_state: str | None = None
         self._attr_unique_id = config_entry.unique_id
         self._attr_is_volume_muted: bool = False
-        self._attr_device_class = MediaPlayerDeviceClass.TV
+        self._attr_device_class = DEVICE_CLASS_TV
```

Comment:
Why you reverted to const ?

Reply:
Rolled back to a commit that leaves `media_player.py` unchanged.